docs/IdentityCard.md

```diff
@@ -0,0 +1,12 @@
+# IdentityCard
+
+- `v::identityCard()`
```

Comment:
I think we can keep it here for a while.

Even if it moves to Locale, the same is too generic as well, an
"IdentityCard" rule must be able to validate any identify card, or at least
more than one.

On Wed, Dec 23, 2015 at 6:19 PM, Tomasz Regdos <notifications@github.com>
wrote:

> In docs/IdentityCard.md
> <https://github.com/Respect/Validation/pull/623#discussion_r48376486>:
>
> > @@ -0,0 +1,12 @@
> > +# IdentityCard
> > +
> > +- `v::identityCard()`
>
> I add proof of concept commit (tests not working).
> Move PESEL and Identity Card to common Poland validator. Do you think that
> is that OK?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Respect/Validation/pull/623/files#r48376486>.
>



-- 
*Henrique Moody*

*Web Developer | ZEND016279*
http://about.me/henriquemoody
*+55 11 98183-2855*